src/convertToObject.js

```diff
@@ -6,7 +6,31 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // if the sourceString is empty return an empty object
+  if (!sourceString) {
+    return {};
+  }
+
+  // remove all tab and new line characters
+  // replaceAll doesn't work on this version of Node
+  const clearedString = sourceString.replace(/[\t\n]/g, '');
+  const stylesArr = clearedString.split(';');
+  const stylesObj = {};
+
+  stylesArr.forEach((style) => {
+    let [name, value] = style.split(':');
+
+    // name and value can be empty or undefined
+    if (name && value) {
+      name = name.trim();
+      value = value.trim();
+      // add a new line character where it is needed
+      value = value.replace(/,/g, ',\n');
```

Comment:
[CODE KNOWLEDGE] - The replacement of commas with commas followed by a newline seems unnecessary for converting the styles to an object and is not mentioned in the task description. It could potentially introduce formatting issues into the values.